.github/workflows/update-translations.yml

```diff
@@ -8,12 +11,30 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+
       - name: Fetch and merge from Weblate
```

Comment:
nit: once these steps are separated by blank lines, I feel like I want the enclosing `jobs:` to be separated by a blank line from its siblings — otherwise it feels like a bit of a visual-hierarchy inversion